drop-cap/library/src/main/java/com/novoda/dropcap/DropCapView.java

```diff
@@ -341,58 +358,54 @@ protected void onDraw(Canvas canvas) {
         }
     }
 
-    private void drawCopyWithoutDropCap(Canvas canvas) {
-        for (int i = 0; i < copyStaticLayout.getLineCount(); i++) {
-            int lineStart = copyStaticLayout.getLineStart(i);
-            int lineEnd = copyStaticLayout.getLineEnd(i);
-
-            CharSequence charSequence = copyStaticLayout.getText().subSequence(lineStart, lineEnd);
-
-            int baseline = copyStaticLayout.getLineBaseline(i) + getPaddingTop();
-
-            canvas.drawText(
-                    charSequence,
-                    0,
-                    charSequence.length(),
-                    getPaddingLeft(),
-                    baseline,
-                    copyStaticLayout.getPaint()
-            );
-        }
-    }
-
     private void drawDropCap(Canvas canvas) {
         float dropCapBaselineFromCopyTop = dropCapBaseline + distanceFromViewPortTop;
-        canvas.drawText(dropCapStaticLayout.getText(), 0, 1, getPaddingLeft(), dropCapBaselineFromCopyTop, dropCapPaint);
+        canvas.drawText(dropCapText, 0, dropCapText.length(), getPaddingLeft(), dropCapBaselineFromCopyTop, dropCapPaint);
     }
 
     private void drawCopyForDropCap(Canvas canvas) {
         for (int i = 0; i < numberOfLinesToSpan; i++) {
-            int lineStart = dropCapStaticLayout.getLineStart(i);
-            int lineEnd = dropCapStaticLayout.getLineEnd(i);
+            int lineStart = dropCapCopyStaticLayout.getLineStart(i);
+            int lineEnd = dropCapCopyStaticLayout.getLineEnd(i);
 
-            int baseline = dropCapStaticLayout.getLineBaseline(i) + getPaddingTop();
-
-            if (i == 0) {
-                lineStart = lineStart + 1;
```

Comment:
This was done to offset the `lineStart` to be after the end of the `DropCap` text but this is easily avoided if the `StaticLayout` only deals with the remaining text.